TramsDataApi/Gateways/TrustGateway.cs

```diff
@@ -46,9 +46,10 @@ public IList<Group> SearchGroups(string groupName, string ukprn, string companie
 
             return _dbContext.Group
                 .Where(g => (
-                    g.GroupName.Contains(groupName) ||
+                    (g.GroupName.Contains(groupName) ||
                     g.Ukprn.Contains(ukprn) ||
-                    g.CompaniesHouseNumber.Contains(companiesHouseNumber)
+                    g.CompaniesHouseNumber.Contains(companiesHouseNumber))
+                    && g.GroupType.Contains("trust")
```

Comment:
Is this case sensitive? I.e. Will this match `"Trust"` as the group name

Reply:
@danielburnley , in the Integration Test I built some of them with a capitalised 'T' and it still passes. This should be fine?

            var groups = Builder<Group>.CreateListOfSize(15)
                .TheFirst(2)
                .With(g => g.GroupType = "Trust")
                .TheNext(2)
                .With(g => g.GroupType = "Single-academy trust")
                .TheRest()
                .With(g => g.GroupType = "Multi-academy trust")
                .All()
                .With(e => e.Ukprn = _randomGenerator.Int().ToString())
                .All()
                .With(e => e.Ukprn = _randomGenerator.Int().ToString())
                .Build()
                .ToList();

Reply:
`Contains` maps to `LIKE` in SQL, so should be case insensitive

Reply:
@nouriach might be worth explicitly checking that with an integration test though